### PR TITLE
Add irssi

### PR DESCRIPTION
**Pull request by alt219**
Added Irssi, a terminal-based IRC client (sans Perl support).

### CODE REVIEW

**Comment by alt219** [comment]
Not mandatory per se – they're mostly help files and colorschemes.  I consolidated them per your suggestion.

**Comment by alt219** [comment]
Also fixed the PATH environment variable per your comment as all.

File: cross/irssi/PLIST [hunk 1]
```diff
@@ -0,0 +1,6 @@
+bin:bin/irssi
+etc:etc/irssi.conf
+shr:share/doc/irssi
+shr:share/irssi/help
```
Comment (Dr-Bean) [inline cross/irssi/PLIST]: We generally don't include helpfiles and docs to minimize package size. Could you make sure that applies here as well?
You can link to outside documentation via `HELP_URL` in spk/Makefile.

File: spk/irssi/Makefile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+SPK_NAME = irssi
+SPK_VERS = 0.8.17
+SPK_REV = 1
+SPK_ICON = src/irssi.png
+
+DEPENDS = cross/irssi
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = Irssi is a terminal based IRC client for UNIX systems. It also supports SILC and ICB protocols via plugins.
+RELOAD_UI = no
```
Comment (Dr-Bean) [inline spk/irssi/Makefile]: Please remove, `no` is the default value.

File: spk/irssi/Makefile [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+DISPLAY_NAME = Irssi
+STARTABLE = no
```
Comment (Dr-Bean) [inline spk/irssi/Makefile]: Please remove, `no` is the default value.

File: spk/irssi/Makefile [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+BETA = 0
```
Comment (Dr-Bean) [inline spk/irssi/Makefile]: `1`

File: spk/irssi/Makefile [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+HOMEPAGE   = http://www.irssi.org/
+LICENSE    = GPLv2
+
+INSTALLER_SCRIPT = src/installer.sh
+SSS_SCRIPT       = src/dsm-control.sh
+ADDITIONAL_SCRIPTS =
```
Comment (Dr-Bean) [inline spk/irssi/Makefile]: Please remove if unused...we should do that for all packages at some point...

File: spk/irssi/src/dsm-control.sh [hunk 6]
```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+# Package
+PACKAGE="irssi"
+DNAME="Irssi"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:/usr/local/bin:/bin:/usr/bin:/usr/syno/bin"
```
Comment (Dr-Bean) [inline spk/irssi/src/dsm-control.sh]: Use ${PATH} here as well.

**Comment by Dr-Bean** [comment]
The icon could use a resize here too, but I'll slap a different label on it already.

**Comment by Dr-Bean** [comment]
Nice work :) 
Fixed download site with 96242e03579749c0fa2226d3be9c7d322dc12bbb, and a slight adjustment to irssi and links with df7009def2689b144b7d48d2d3ad29182e1f1279 (package shows as stopped otherwise...not that it matters, but still ;))

**Pull request merged**